docs/install.rst

```diff
@@ -34,12 +34,11 @@ In production environments, always activate Django template cache loader. This i
 
 Template packs
 ~~~~~~~~~~~~~~
+Since version 2.0, django-crispy-forms has built-in support for version 3 and 4 of the Bootstrap CSS framework
```

Comment:
This could be part of a separate commit but wanted to make the change now so I don't miss it. I've not yet removed the uni-form package but it's on the to do list before v2.0